src/aliases.jl

```diff
@@ -38,6 +38,7 @@ function n_rays end
 function n_vertices end
 
 function number_of_partitions end
+function number_of_weak_compositions end
 function number_of_patches end
```

Comment:
```suggestion
function number_of_patches end
function number_of_weak_compositions end
```
keep the list sorted